spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/config/EnvironmentRepositoryConfiguration.java

```diff
@@ -301,6 +317,19 @@ public NativeEnvironmentRepository nativeEnvironmentRepository(
 
 }
 
+@Configuration
+@Profile("s3")
```

Comment:
I would be changing the name to just `aws`

Reply:
I don't know. They have a property store that could have an impl in the future.

Reply:
Maybe 'awss3'

Reply:
maybe `awss3`?  `s3` just seems a little vague

Reply:
`awss3` it is then

Reply:
fixed